localization/react-intl/src/app/components/UserInvalidInvitation.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "id": "userInvalidInvitation.heading",
+    "defaultMessage": "Failed to accept your invitation"
+  },
+  {
+    "id": "userInvalidInvitation.message",
+    "defaultMessage": "Sorry could not accept your invitation, may be your invitaion expired or team not exists."
```

Comment:
"An error occurred while processing your invitation. Please make sure the invitation date is still valid and that the team still exists. Contact the support team if the issue persists."